python-backend/app/api/variances/resources/variance_resource.py

```diff
@@ -55,7 +56,7 @@ def _apply_filters_and_pagination(self,
             status_filter_values = application_status.split(',')
 
         filtered_query = apply_filters(
-            Variance.query,
+            Variance.query.order_by(desc(Variance.received_date)),
```

Comment:
Oh, we should add to the Swagger docs that the sort order is `received_date`

Reply:
Can that be done in this PR?

Reply:
added!